From: Stephane Glondu <steph@glondu.net>
Date: Fri, 21 May 2010 15:53:08 +0200
Subject: Do not remove ocamlbuild.Makefile

This file is present in upstream tarball; do not remove it so that the
tree is in the same state as freshly unpacked after distclean.

Signed-off-by: Stephane Glondu <steph@glondu.net>
---
 ocamlbuild.Makefile |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ocamlbuild.Makefile b/ocamlbuild.Makefile
index ef32e7d..f8ee8ec 100644
--- a/ocamlbuild.Makefile
+++ b/ocamlbuild.Makefile
@@ -214,4 +214,4 @@ cleaner:: clean
 	make -C examples clean
 
 dist-clean distclean:: clean
-	rm -f Makefile config.cache config.log config.status META version.ml myocamlbuild.ml ocamlbuild.Makefile simple.Makefile
+	rm -f Makefile config.cache config.log config.status META version.ml myocamlbuild.ml simple.Makefile
-- 
